.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
+encoding//lib/galaxy/datatypes/text.py=utf-8
```

Comment:
Do we need this file?

Reply:
No. I meant to remove it, thanks for reminder! Not sure how it got committed, thought i had added it to gitignore.